Makefile

```diff
@@ -1,7 +1,9 @@
-PYTHON = python
 TEST_PATH = ./tests/
 FLAKE8_EXCLUDE = venv,.venv,.eggs,.tox,.git,__pycache__,*.pyc
-SHELL := /bin/bash
+PROJECT = pykg2tbl
+AUTHOR = Marc Portier, Cedric Decruw 
+
+.PHONY: build docs format install jupyter publish run tests
```

Comment:
phony keys don't match with the ones below: e.g. test vs tests, jupyter, ...

Reply:
we really should make a standard for these stuff